exchanges/bybit/bybit_wrapper.go

```diff
@@ -1652,8 +1652,8 @@ func (e *Exchange) UpdateOrderExecutionLimits(ctx context.Context, a asset.Item)
 		return fmt.Errorf("%s %w", a, asset.ErrNotSupported)
 	}
 	l := make([]limits.MinMaxLevel, 0, len(allInstrumentsInfo.List))
-	for x := range allInstrumentsInfo.List {
-		symbol := allInstrumentsInfo.List[x].transformSymbol(a)
+	for _, info := range allInstrumentsInfo.List {
```

Comment:
🐒 Optional nitpick:
`info` is not a good variable name for this; Everything is information.
```suggestion
	for _, inst := range allInstrumentsInfo.List {
```

Please don't hate me.